package.json

```
@@ -75,6 +77,11 @@
       "default": true,
       "description": "Use an alternative scoring approach which prefers run of consecutive characters, acronyms and start of words. (Experimental)"
     },
+    "useRipGrep": {
+      "type": "boolean",
+      "default": false,
+      "description": "Use the experimental `ripgrep` crawler. This will speed up substantially the indexing process on large projects."
```

Minor phrasing suggestion: I think this sounds more natural...

```suggestion
      "description": "Use the experimental `ripgrep` crawler. This will substantially speed up the indexing process on large projects."
```

---

haha! thanks for pointing it out 😅